<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botonCalError">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>95</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cálculo de Errores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonSistNum">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>123</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sistemas de numeración</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonPuntoFlot">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>77</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Punto flotante</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonPropError">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Propagación de errores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonBolzano">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>105</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Teorema de Bolzano</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonBiseccion">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>105</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Método de Bisección</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonSalir">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
